### PR TITLE
Equalize all uniforms in terms of style and a few other cases

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request
Gives all uniforms STYLE_LOW (1), changes some description cases, removes cheap tuxedo (replaces it with a normal tuxedo), removes few biased cases (cat ears, clown poncho lol), removes those high style uniform prices

## Why It's Good For The Game
The current style designation goes a little against the whole idea of style (*in my opinion*) - it's reasonable that we steered from the old doc given introduction of style dodge, but given how much uniforms literally given you nothing but the looks I don't want to fucking operatives to switch to that powergamish helltaker attire instead of their uniforms which fit with their armor.

## Changelog
:cl:
balance: All uniforms now have STYLE_LOW (1)
balance: cat ears and clown poncho don't have -2 style
balance: lowered (practically removed) all in-code uniform prices, which means they'll now appear in trash
del: cheap tuxedo, replaced with normal tuxedo
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
okay why trhe fuck it conflicts

**Comment by ErisMeleeEnjoyer** [comment]
normal tuxedo is a +2 style item, shouldn't be in loadout
.
.
.
according to humon, that is. I don't care. Based, even.

**Comment by Firefox13** [comment]
You didn't even look at the code.

**Comment by MLGTASTICa** [comment]
what about paramedic hardsuit having style ? i get that its a hardsuit but it provides universally no armor and it looks kinda banger ngl

File: code/modules/clothing/head/collectable.dm [hunk 1]
```diff
@@ -104,7 +104,6 @@
 	desc = "The fur feels... a bit too realistic."
 	icon_state = "kitty"
 	body_parts_covered = 0
-	style = STYLE_NEG_HIGH//no.
```
Comment (Humonitarian) [inline code/modules/clothing/head/collectable.dm:107]: This marks the downfall of Eris.

**Comment by Firefox13** [comment]
I will address other slot changes in other PRs because it's a rather finnicky balance question.

**Comment by ErisMeleeEnjoyer** [comment]
> You didn't even look at the code.

yes i didn't 
and

File: code/modules/clothing/head/collectable.dm (hunk 1 above)
Comment (Firefox13) [reply code/modules/clothing/head/collectable.dm:107]: https://youtu.be/aAYkdGmSe38?t=90

**Comment by Humonitarian** [comment]
All of these changes are 10/10 and have my complete delegate approval

Two other proposed changes for further PRs include uniforming protective coats (IH coat is peak powergame, by design, but might need further rediscussion), and light armor providing only -1 style (nonfullbody vests and simple helmets)

**Review by SirRichardFrancis — COMMENTED** [review]
Might as well drop the price on jumpsuits in style vendor.

**Comment by Firefox13** [comment]
Will do this in another PR too.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -389,29 +354,25 @@ obj/item/clothing/under/genericr
 	desc = "A suit and pants that are so basic it's stylish."
```
Comment (Firefox13) [inline code/modules/clothing/under/miscellaneous.dm:354]:
```suggestion
	desc = "A white satin shirt that can really catch the light and flare-cut trousers."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -389,29 +354,25 @@ obj/item/clothing/under/genericr
...
 	icon_state = "jamrock_suit"
 	item_state = "jamrock_suit"
-	style = STYLE_HIGH
 
 /obj/item/clothing/under/aerostatic//Elysium
 	name = "dark comfortable clothing"
 	desc = "A suit and pants that are so basic it's stylish."
```
Comment (Firefox13) [inline code/modules/clothing/under/miscellaneous.dm:360]:
```suggestion
	desc = "A tan shirt and brown pants that are rather basic."
```

File: code/modules/client/preference_setup/loadout/lists/uniforms.dm [hunk 4]
```diff
@@ -163,8 +163,8 @@
 	gear_tweaks += new /datum/gear_tweak/path(generic)
 
 /datum/gear/uniform/tuxedo
-	display_name = "cheap tuxedo"
-	path = /obj/item/clothing/under/tuxedo/cheap
+	display_name = "tuxedo"
+	path = /obj/item/clothing/under/tuxedo
 	cost = 2
```
Comment (Firefox13) [inline code/modules/client/preference_setup/loadout/lists/uniforms.dm:168]:
```suggestion
	cost = 1
```

**Review by SirRichardFrancis — APPROVED** [review]

**Review by Pink-Chink — APPROVED** [review]

**Pull request merged**